config/crd/bases/policies.kubewarden.io_policyservers.yaml

```diff
@@ -173,7 +173,7 @@ spec:
                 description: Replicas is the number of desired replicas.
                 format: int32
                 type: integer
-              security:
+              securityContexts:
```

Comment:
I'm fine with that. But I think that @viccuad suggested to use the plural word because we have 2 `securityContext` in this field. One for pod and another for container.

Reply:
Should I still change to `securityContext`?

Reply:
No sorry, it's my fault. The new name is perfect because of the two different security context objects being hold by this struct.